server/itemAlreadyExists.ts

```diff
@@ -0,0 +1,14 @@
+"use server";
+
+import prisma from "@/lib/prisma";
+
+export async function itemAlreadyExists(
+  item: string,
+  pricePerUnit: number,
+  storeID: string
+) {
```

Comment:
be good to get into the habit of defining the return type of your function, there are ways to force you to specify but we haven't set it up in this project

Reply:
https://github.com/amiaoh/groceryDatabaseApp/pull/30/commits/9663a8ffd187cf6ddc9e8e7ff6ff1a12b9ab1e97

Reply:
https://github.com/amiaoh/groceryDatabaseApp/pull/30/commits/d7f75f437bf6089b4a5c78786bc7be1752bb511a

Reply:
also used shortcut syntax

Reply:
you only linked the shortcut syntax commit but I see you've added the return type as well 👍